qemu/tests/cfg/fwcfg.cfg

```diff
@@ -19,21 +19,17 @@
     image_name_stg = "images/storage"
     remove_image_stg = yes
     no ppc64 ppc64le aarch64
+    chk_windbg_cmd = 'wmic product get name,version | find /i "SDK Debuggers"'
```

Comment:
As WMIC would be deprecated gradually, it is superseded by windows powershell for WMI.

Reply:
OK, I will find the replacement command.

Reply:
This command line could be a replacement.
```
Get-WmiObject Win32_Product | Where-Object Name -like "*SDK Debuggers*" | Select Name, Version
```